src/devcards.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DevCards from './components/DevCards.jsx';
```

Comment:
I don't generally bother making the distinction between JS and JSX files - as they run through the same compiler config anyway.